tests/all/test_builder.rs

```diff
@@ -695,7 +698,7 @@ fn test_bitcast() {
     let first_iv = cast.as_instruction_value().unwrap();
 
     builder.position_before(&first_iv);
-    builder.build_bitcast(f32_arg, i64_type, "f32toi64");
+    builder.build_bitcast(f64_arg, i64_type, "f64toi64");
 
-    assert!(module.verify().is_err());
+    assert!(module.verify().is_ok());
```

Comment:
I think the point of this was to test that LLVM will infact declare the module invalid if you try to bitcast `f32` to `i64`. So long as it isn't UB, what's the issue with this?

Reply:
An asserts build of LLVM will assert and terminate the program.

It's true that creating the bitcast instruction won't trigger UB internally to LLVM, but there is no way to ever convert such a bitcast into code that in valid. The type of a value is immutable assigned only upon creation, so neither the target type of the bitcast nor the type of its operand can change. LLVM has a similar assertion for RAUW between different types too.